discussion-questions.md

```diff
@@ -1,8 +1,15 @@
 #  How much time did you spend on this assignment?
 
-_replace this text with your answer_
+The time is estimated.
+My computer had lots of problems in setting up the IntelliJ program. It probably took me at least 3 hours to do that.
+
+Adding fluency to the picture program and practicing scala functions is quite easy. I spend at most 2.5 hours.
 
 # Discussion questions / topics
 Below, add one or two things that you might want to discuss when we meet again.
 
-_replace this text with your discussion questions / topics_
\ No newline at end of file
+-we can talk more about how to implement a DSL. Are we doing a DSL in modifying the Picture program?
+
+-more practise in using Scala. I still don't feel comfortable in using Scala. I definitely need more practise in it. 
+
+-explain more about "fluency". Does "fluency" means making the program shorter? or it means to make the program easier to read?
```

Comment:
I hope we answered some of your questions in our most recent group session!

Regarding Scala: I certainly don't expect you to feel comfortable using Scala yet! Learning a new language takes time and practice, as you mentioned :).